pages/linux/apt.md

```diff
@@ -1,6 +1,7 @@
 # apt
 
 > Package management utility for Debian based distributions.
+> Recommended replacement for apt-get in Ubuntu versions 16.04 and later.
```

Comment:
I think it's more useful in the `apt-get` page. Because people searching for `apt` are already likely to use the `apt` command. So its more helpful to redirect them from `apt-get` to `apt`. But `apt-get` already has 2 lines. So I doubt whether we will be able to fit this in somewhere.

Reply:
The idea behind putting that up was that I had people asking me about the difference.

Reply:
Sure, in that case, they can just do `tldr apt-get` and know the difference.